src/pages/jp/index.js

```diff
@@ -19,11 +19,13 @@ import Button from '../../components/button'
 const NormalBox = withNormalHelpers(Box)
 
 const caseLogos = [
-  'unext',
   'square',
+  'unext',
+  'colopl-ja',
   'lenovo',
   'tencent',
   'bigo',
+  'colopl-ja',
```

Comment:
Do you need to add two same colopl logos on this page?

Reply:
Fixup